app/views/distributions/edit.html.erb

```diff
@@ -24,7 +24,7 @@
 
 <section class="content">
 
-  <% unless @distribution.future? %>
+  <% if @distribution.issued_at && !@distribution.future? %>
```

Comment:
I'm a bit confused here. If we just added a validation ensuring that `issued_at` can't be null, why do we need this check on an edit form?

Reply:
Yea so previously there was no way to render a view with `issued_at` having a value of `nil`, because even if you tried to update it to be blank, a before_save callback would convert the blank value into a value based on `created_at`. That meant that these two instance methods (`future?` and `past?`) were always safe to use. 

After this change `issued_at` can't be persisted to the db as `nil` but you can try to update it to be `nil`, then when you try to save the validations will fail, and the edit page will be rendered (with your old parameters) and trying to render the view with `nil` for `issued_at` blows up here with a `NoMethodError # no method > in NilClass`.

Reply:
I thought about moving this presence guard clause into the `Distribution#future?` method. But I feel like that might lead to unexpected behavior, because returning either false or true for `future?` doesn't really seem right to me.